app/views/dashboard/index.html.erb

```diff
@@ -8,6 +8,9 @@
         <li class="nav-item">
           <a class="nav-link" href="/orders">My Orders</a>
         </li>
+        <li class="nav-item">
+          <a class="nav-link" href="/user/stores">My Stores</a>
```

Comment:
I notice that the path for Orders is /orders and we scope the orders through the current user.

What's the reasoning for nesting the stores in users instead of doing '/stores' and scoping the stores through the user?

Reply:
This was namespaced to reserve the "/stores" url for a stores index page that includes all stores in our site.

Reply:
Gotcha. works for me!